source/vibe/data/json.d

```diff
@@ -44,7 +44,7 @@ unittest {
 public import vibe.data.serialization;
 
 public import std.json : JSONException;
-import std.algorithm : equal, min;
+import std.algorithm : equal, min, max;
```

Comment:
Do you suggest to move `std.algorithm : max` inside struct Json?
I can't move/remove `import std.algorithm : equal, min` because it irrelevant with this PR.